File: ExampleApp/src/ScriptNoise.cs
using Prism;

namespace Example
{
    public class ScriptNoise : Behaviour
    {
        private void OnCreate()
        {
            var shader = PrismShader.GetShader("Custom/NoiseTest");
            var material = Material.Create(shader);
            var meshComponent = GetComponent<MeshRendererComponent>();
            meshComponent.SetMaterial(0, material);
        }
        private void OnUpdate()
        {

        }
    }
}

File: PrismEditor/Assets/scripts/Python/ScriptNoise.py
from PrismEngine import Behaviour, Log, MeshRendererComponent, Material, PrismShader


class ScriptNoise(Behaviour):
    def OnCreate(self):
        shader = PrismShader.GetShader("Custom/NoiseTest")
        material = Material(shader)
        meshComponent = self.GetComponent(MeshRendererComponent)
        meshComponent.SetMaterial(0, material)

    def OnUpdate(self):
        pass
